src/box/tuple.h

```diff
@@ -31,9 +31,8 @@
  * SUCH DAMAGE.
  */
 #include "trivia/util.h"
-#include "say.h"
 #include "diag.h"
-#include "error.h"
+#include "memtx_block.h"
```

Comment:
I don't like that `tuple.h` includes `memtx_block.h`. Maybe, we could name the file in a more generic way? For example, "compact size" or something like this?